### PR TITLE
JSWeekHomework

### PR DESCRIPTION
**Pull request by CharlesKimMaina**

### CODE REVIEW

File: javascript/javascript2/week3/delayCallApp.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+        function logOut(delay, stringToLog) {
```
Comment (Rassibassi) [inline javascript/javascript2/week3/delayCallApp.html:10]: good job! But please make the correct the indentation, it's not very pleasant to read code with the wrong format

File: javascript/javascript2/week3/functionAsVariable.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+const arrayFunc = [
+  function functionname() {},
+
+  function functionname2() {},
+
+  function functionname3() {}
+];
+
+let arrayIteration = arrayFunc.map((individualFunct)=>{
```
Comment (Rassibassi) [inline javascript/javascript2/week3/functionAsVariable.html:19]: can be `const`

File: javascript/javascript2/week3/functionAsVariable.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+
+additionFunct();
+addFunc();
+
+const object={
+  addSum: function addFunc(){},
+}
+  console.log(object.addSum);
```
Comment (Rassibassi) [inline javascript/javascript2/week3/functionAsVariable.html:40]: I dont think you have 100% understood what you should do here. inside the object you redefine a function. Can you just reuse the one already defined in line 30. Here I would also recommend to define functions as variables:

`const myFunctionName = function(){...}`

File: javascript/javascript2/week3/planetLogger.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+        const earthLogger = function() {
```
Comment (Rassibassi) [inline javascript/javascript2/week3/planetLogger.html:10]: Watch the indentation!

File: javascript/javascript2/week3/delayCallApp.html (hunk 1 above)
Comment (CharlesKimMaina) [reply javascript/javascript2/week3/delayCallApp.html:10]: I have downloaded Prettier,.now my codes are well indented, Thanks

File: javascript/javascript2/week3/functionAsVariable.html (hunk 2 above)
Comment (CharlesKimMaina) [reply javascript/javascript2/week3/functionAsVariable.html:19]: Is changed to const, Thanks

**Pull request merged**